components/sso-saml/org.wso2.carbon.identity.sso.saml.common/src/main/java/org/wso2/carbon/identity/sso/saml/common/Util.java

```diff
@@ -206,6 +220,7 @@ private static String normalizeUrlEncoding(String text) {
                     else
                         normalized.append(percentCode);
                 } catch (UnsupportedEncodingException e) {
+                    log.error("Url Encoding not supported.", e);
```

Comment:
this should be debug log because exception has been ignored in the original code.